src/main/java/ua/goit/timonov/enterprise/module_6_2/view/menus/EmployeesHandler.java

```diff
@@ -0,0 +1,86 @@
+package ua.goit.timonov.enterprise.module_6_2.view.menus;
+
+import ua.goit.timonov.enterprise.module_6_2.controllers.EmployeeController;
+import ua.goit.timonov.enterprise.module_6_2.model.DbItem;
+import ua.goit.timonov.enterprise.module_6_2.model.Employee;
+import ua.goit.timonov.enterprise.module_6_2.view.console.ConsoleIO;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Handler for tasks with DB Restaurant's component Employee
+ * with implementation of methods from DbItemHandler:
+ * - get from DB list of all employees
+ * - add new employee to DB
+ * - search employee by ID
+ * - search employee by full name (surname & name)
+ * - delete employee from DB by ID
+ * - search employee from DB by full name (surname & name)
+ */
+public class EmployeesHandler extends DbItemHandler {
+
+    public static final String EMPLOYEE = "Employee";
+
+    /**
+     * Uses inherited constructor with setting component's name
+     */
+    public EmployeesHandler() {
+        super(EMPLOYEE);
+    }
+
+    private EmployeeController employeeController;
+
+    public void setEmployeeController(EmployeeController employeeController) {
+        this.employeeController = employeeController;
+    }
+
+    // implementation of inherited methods from DbItemHandler
+    @Override
+    protected List<DbItem> getAllItems() {
```

Comment:
fixed using Generics instead DbItems in classes EmployeeHandler, DishHandler etc.